app/templates/_webpack.config.js

```diff
@@ -0,0 +1,39 @@
+const webpack = require('webpack');
+
+module.exports = {
+	entry: './src/<%= componentName %>.js',
+	module: {
+		rules: [{
+			test: /\.js$/,
+			exclude: /(node_modules)/,
+			use: {
+				loader: 'babel-loader',
+				options: {
+					compact: false,
+					presets: ['babel-preset-es2015'],
+					plugins: ['babel-plugin-transform-node-env-inline']
+				}
+			}
+		}, {
+			test: /\.scss$/,
+			use: [{
+				loader: "style-loader"
+			}, {
+				loader: "css-loader"
+			}, {
+				loader: "sass-loader"
+			}]
+		}]
+	},
+	output: {<% if (buildFormat === 'amd') { %>
+		library: '<%= repoName %>/src/<%= componentName %>',
+		libraryTarget: '<%= buildFormat %>',
+		filename: './build/<%= buildFormat %>/<%= repoName %>/src/<%= componentName %>.js'<% } else { %>
+		library: 'metal',
+		libraryTarget: 'this',
+		filename: './build/<%= buildFormat %>/<%= kebabCaseName %>.js'<% } %>
+	},
+	plugins: [
+		new webpack.optimize.ModuleConcatenationPlugin()
```

Comment:
What if we add the [resolve alias](https://github.com/metal/metal-clay-components/blob/master/packages/clay-button/webpack.config.js#L28) in the generator? The highest component will resolve dependencies correctly. With the documentation you've suggested, @jbalsas, developers who didn't use the generator will be able to a fix his setup.